src/handlers/swap_handlers.ts

```diff
@@ -377,7 +377,6 @@ export class SwapHandlers {
                     },
                 ]);
             }
-            req.log.info('Uncaught error', e.message, e.stack);
```

Comment:
This is not necessary as we already re-throw the error and information about the error (and stack trace) will be printed out by the errorHandler middleware here: https://github.com/0xProject/api-utils/blob/69dd016bc3e1a97b27b2a19936603334c71562c3/src/middleware/error_handler.ts#L107